Assets/JavaScript/booksList-3.js

```diff
@@ -32,8 +32,7 @@ class AwsomeBooks {
       .map(
```

Comment:
- [ ] [optional] @Team great work on your app, however, it would be nice if when clicking add a new book, the input field would be cleared of data entered and maybe a message displayed of successfully added a book.